src/pages/EditResourcePage.tsx

```diff
@@ -85,22 +90,63 @@ const EditResourcePage: FC = () => {
 
       getResourceDefaults(startingResource.identifier).then((responseWithCalculatedDefaults) => {
         saveCalculatedFields(responseWithCalculatedDefaults.data);
-        setFormikInitResource({
-          ...deepmerge(startingResource, responseWithCalculatedDefaults.data),
-          contributors: [
-            {
-              identifier: contributorResponse.data.identifier,
-              features: {
-                dlr_contributor_identifier: contributorResponse.data.identifier,
-                dlr_contributor_name: user.institution,
-                dlr_contributor_type: contributorFeatureNames.Institution,
+        if (
+          responseWithCalculatedDefaults.data.creators &&
```

Comment:
Den første sjekken trengs kanskje ikke om du skriver responseWithCalculatedDefaults.data.creators?.[0] på de to neste sjekkene